generator/lib/dockerfile.py

```diff
@@ -8,7 +8,7 @@
 
 class Dockerfile:
     def __init__(self, from_image, commands):
-        self.from_image = "FROM {}\n".format(from_image)
+        self.from_image = "FROM {}\nARG TARGETARCH\n".format(from_image)
```

Comment:
instead of polluting the from_image variable, append it directly to the `output` string of `to_string` function

https://github.com/gardener/ops-toolbelt/blob/eb2a596125c6d51db335d1cbfcdfeabc3044cd99/generator/lib/dockerfile.py#L27-L31

Reply:
Done with d645abbb55af716512c21706ac831444d41a0cbc